docs/content/_docs/query_language.html

```diff
@@ -284,12 +242,31 @@ <h4>Operators</h4>
     <td><pre><code class="language-hql">&lt;a&gt; not in [&lt;b&gt;, ..]</code></pre></td>
     <td><pre><code class="language-json">["not", ["or", ["=", &lt;a&gt;, &lt;b&gt;], ..]]</code></pre></td>
   </tr>
+  
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">&lt;a&gt; true [&lt;b&gt;, ..]</code></pre></td>
```

Comment:
`true` and `false` don't seem to apply to tags, they're just static filters [which always return those values](https://github.com/spotify/heroic/blob/master/heroic-component/src/main/java/com/spotify/heroic/filter/TrueFilter.java#L47)

Reply:
was debating if i should document them or not in this case